web/src/app/projects/repository/repository.component.ts

```diff
@@ -29,23 +30,17 @@ export class RepositoryComponent implements OnInit {
       .findOneById(this.uid)
       .subscribe((repository: RepositoryModel) => {
         this.repository = repository;
-        this.sortMilestones();
+        if (this.repository.milestones) {
+          this.sortingService.sortList(this.repository.milestones, 'updatedAt');
+        }
+        if (this.repository.releases) {
+        this.sortingService.sortList(this.repository.releases, 'createdAt');
```

Comment:
Indentation is incorrect here, but can come in the next PR

Reply:
Will fix it